app/controllers/forms_controller.rb

```diff
@@ -30,6 +30,6 @@ def mark_pages_section_completed
 private
 
   def mark_complete_input_params
-    params.require(:forms_mark_complete_input).permit(:mark_complete).merge(form: current_form)
+    params.require(:forms_mark_pages_section_complete_input).permit(:mark_complete).merge(form: current_form)
```

Comment:
This line wasn't updated initially, so marking the pages section complete threw an error